docs/development_guide.md

```diff
@@ -101,8 +138,12 @@ git push -u origin <branch name>
 To commit and push those changes to your branch.
 
 ### Syncing your branch
-If you want to pull in changes from the target branch (i.e. googlegenomic:master),
-run:
+To update the forked repository, you may follow instructions on [syncing a fork](https://help.github.com/articles/syncing-a-fork/), or simply set upstream to the main repository and do:
+```bash
```

Comment:
Do you need this section? This is the same as the instructions below except it doesn't have --rebase (and I think we needed to always pull with rebase for things to work smoothly).

Reply:
I am thinking there are two scenarios. 
Before checking out a new branch from the forked master branch, we would like to keep the forked master branch to be synced with the remote master branch, so that the commits from other people will apply. In that case, we update the forked master repository.
The second portion is after we made changes to local branch, we pull in the latest commits, then replay our local changes.
But you are right, the second portion is sufficient for the sync. I will remove this part.